feature_layers/feature-collection-layer-from-portal/README.md

```diff
@@ -0,0 +1,34 @@
+# Feature Collection Layer From Portal
+
+This sample demonstrates opening a feature collection saved as a portal item
+
+![](FeatureCollectionLayerFromPortal.png)
+
+## Use case
+
+Feature collection layers are often used to hold features with mixed geometry or unstructured data. You can display feature collections stored in a Portal, which are often used for showing content from a CSV or map notes.
+
+## How to use the sample
+
+The feature collection is loaded from the Portal item when the sample starts. Enter feature collection saved as portal item in the text field
+and click on button to see result.
+
+## How it works
+
+To display a `FeatureCollection` as a `portal Item` on an `ArcGISMap` :
```

Comment:
We've decided in the readme review process to gemerally not start these sections with a sentence such as 'To do X...', so we could just remove this.
```suggestion
```